.github/workflows/testing.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.10", "3.11", "3.12"]
+        python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
For Python 3.13, there is no wheel for dscribe that makes CI bit longer.

🔗. https://github.com/himkt/pymatgen-analysis-defects/actions/runs/18054940932/job/51383430973?pr=1#step:4:484